activation/listeners.go

```diff
@@ -35,3 +36,20 @@ func Listeners(unsetEnv bool) ([]net.Listener, error) {
 	}
 	return listeners, nil
 }
+
+// Create TLS-ready TCP listeners
+func ListenersTls(unsetEnv bool, tlsConfig *tls.Config) ([]net.Listener, error) {
+	listeners, err := Listeners(unsetEnv)
```

Comment:
Pedantic, but I would rather explicitly return `nil, err` here if Listeners() returns nil, rather than relying on it implicitly returning nil to us